hypothesis-python/tests/cover/test_slippage.py

```diff
@@ -218,3 +218,16 @@ def test(i):
 
     with pytest.raises(MultipleFailures):
         test()
+
+
+def test_finds_multiple_failures_in_generation():
+    @settings(database=None)
+    @given(st.lists(st.floats()))
+    def test(x):
+        mean = sum(x) / len(x)  # ZeroDivisionError
+        if len(x) == 1:
+            assert mean == x[0]  # x[0] is nan
+        assert min(x) <= mean <= max(x)  # x contains nan or +&- inf
+
+    with pytest.raises(MultipleFailures):
```

Comment:
I think this test needs to turn shrinking off in order to reliably guarantee that it's testing what it means to.

Reply:
Shrinking disables the continuing search though!  Without continuing search, this finds the ZeroDivisionError with an empty list, and stops at the first example.